.travis.yml

```diff
@@ -127,7 +131,9 @@ before_install:
      #  Force ARGS to correct prefix, etc for docker image
      export ARGS="--prefix=/usr --sysconfdir=/etc --with-systemdsystemunitdir=/etc/systemd/system --localstatedir=/var --with-flux-security --enable-caliper"
   fi
-
+ - docker pull minio/minio
```

Comment:
It doesn't hurt anything, but you shouldn't need the docker pull minio/minio here since the travis builder will start with an empty docker image cache. (i.e. this line could be removed)